picard/script.py

```diff
@@ -1280,11 +1326,10 @@ def func_datetime(parser, format=None):
     try:
         return datetime.datetime.now(tz=local_tz).strftime(format)
     except ValueError:
+        stackitem = parser._function_stack.get()
         raise ScriptRuntimeError(
-            "Unsupported format code",
-            parser._function_stack.get(),
-            parser._y,
-            parser._x
+            stackitem,
+            "Unsupported format code"
```

Comment:
Since it is shorter now, it can be on one line.

```python
raise ScriptRuntimeError(stackitem, "Unsupported format code")
```

Reply:
All on one line now.